### ISSUE TITLE
exception on sparky src.dest chain

### ISSUE THREAD

**Issue by Zaitsev** [fuse-box/fuse-box#1986]
on Windows I've got ` UnhandledPromiseRejectionWarning:`
```javascript
    await src('./src/assets/**/*.ttf')
        .dest('./dist/renderer/assets','./src/assets')
        .exec()
    ;
```
to fix it  [newLocationBase](https://github.com/fuse-box/fuse-box/blob/3c04a72bdd544c5537402e6f49ad167c98a82232/src/sparky/sparky_chain.ts#L46) should be `path.normalize`-d

```javascript
const s = path.normalize(utils_1.ensureFuseBoxPath(file)).split(path.normalize(newLocationBase));
```

**Comment by nchanged** [comment]
@Zaitsev can you PR it?

**Comment by charsleysa** [comment]
Closed by #1987.
